File: app/src/main/java/com/keelim/hardware/GpslocActivity.java
package com.keelim.hardware;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;


public class GpslocActivity extends Activity{
    
   TextView tr,ts;
   double latitude,longitude;
    
    @Override
    protected void onCreate(Bundle savedInstanceState) {
    
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_gpsloc);
        GpslocateActivity gps = new GpslocateActivity(this);
        latitude = gps.getLatitude();
        longitude = gps.getLongitude();
        tr= findViewById(R.id.textView60);
        ts= findViewById(R.id.textView61);
        
    }

	@Override
	protected void onStart() {
		tr.setText("latitude "+ latitude);
		ts.setText("longitude "+ longitude);
		super.onStart();
	}
    
    }

File: app/src/main/java/com/keelim/hardware/activity/GpslocActivity.kt
package com.keelim.hardware.activity

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.keelim.hardware.R
import kotlinx.android.synthetic.main.activity_gpsloc.*

class GpslocActivity : AppCompatActivity() {
    private var latitude = 0.0
    private var longitude = 0.0
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_gpsloc)
        val gps = GpslocateActivity(this)
        latitude = gps.getLatitude()
        longitude = gps.getLongitude()
    }

    override fun onStart() {
        super.onStart()
        gps_tv1!!.text = "latitude $latitude"
        gps_tv2!!.text = "longitude $longitude"
    }
}